apps/mempool/docker-compose.yml

```
@@ -0,0 +1,61 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+  web:
+    image: mempool/frontend:v2.1.0@sha256:ee952d10f6463bff2de21bcceac8ff8f43ba4a5bba2a1aa39901c8a05761a9b9
+    user: "1000:1000"
+    logging: *default-logging
+    restart: on-failure
+    stop_grace_period: 1m
+    command: "./wait-for db:3306 --timeout=720 -- nginx -g 'daemon off;'"
+    ports:
+      - ${APP_MEMPOOL_PORT}:8080
```

This is running on port `8080` on the container but `$APP_MEMPOOL_PORT` (3006) on the host.

This breaks the hidden service which is proxying to `<app-mempool-ip>:<app-mempool-port>` when the app is actually running at `<app-mempool-ip>:8080`.

It should be running on `APP_MEMPOOL_PORT` on the container too:

```suggestion
      - ${APP_MEMPOOL_PORT}:${APP_MEMPOOL_PORT}
```

You'll also need to pass this value to the Node.js process to tell it to run on that port instead of 8080.